frontend/ui-pablo/app/components/Organisms/AllBondTable.tsx

```diff
@@ -54,6 +54,8 @@ export const AllBondTable: React.FC = () => {
 
   useEffect(() => {
     dispatch(addNextDataBondPools({ startIndex }));
+    // Only called once.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
```

Comment:
If it needs to be called only once, you can omit the dependency list instead.

Reply:
If it is removed, it is going to be called on every render, which is not something that we would want in this case.